src/summary.rs

```diff
@@ -62,7 +64,24 @@ impl Summary {
         msg: &Message,
         chat: &Chat,
         contact: Option<&Contact>,
-    ) -> Self {
+    ) -> Result<Summary> {
+        if let Some((reaction_msg, reaction_contact_id, reaction)) = chat
+            .get_last_reaction_if_newer_than(context, msg.get_timestamp())
```

Comment:
`get_timestamp()` returns the "sent" timestamp, but we need the sort timestamp here because it's what actually tells which message is newer

Reply:
probably makes sense, also as that timestamp is more robust wrt wrong incoming data. i was a bit unsure as we're using  "sent" timestamp for comparable `update_timestamp()` calls - but when looking closer, there is a comment for the "last subject timestamp" that explicitly says why "sort timestamp" should be used for user-visible things